### ISSUE TITLE
Review #3: clarify the parameters of the synthetic experiments

### ISSUE THREAD

**Issue by mickymuis** [mickymuis/vouwpapier#7]
> several parameters are not explained clearly. For example, it is mentioned that "both the pattern elements and the noise are picked from the same uniform random distribution ...", however, how does this relate to the specific pattern sizes included? It would be nice to see some more details here.

I can really understand this and I really, really want to elaborate on this, given more space obviously. I hope there is any way we can further clarify the synthetic benchmark without losing more space. Maybe we should look into it after we have fixed the rest and see if we can rephrase things to contain more information.

**Comment by mleeuwen** [comment]
Indeed, we should either incorporate this in the paper, or refer to a longer (and more complete) description of the data generation process in the Appendix on arXiv (and hence replace that one with an updated version)?

You'll need to write this description for your thesis anyway. :-)
